velox/core/PlanNode.h

```diff
@@ -710,6 +712,84 @@ class AggregationNode : public PlanNode {
   const RowTypePtr outputType_;
 };
 
+class TableWriteNode : public PlanNode {
```

Comment:
We only need to add a new ctor and wrap the old ctor with compatibility macro? Thanks!